packages/web-components/src/components/cbp-card/cbp-card.tsx

```diff
@@ -47,56 +47,31 @@ export class CbpCard {
   }
 
   render() {
-    return (
-      <Host>
-        {this.interactive === 'clickable' && 
+    if(this.interactive === 'clickable'){
+      return(
+        <Host>
           <a class='cbp-card-clickable' href={this.href}>
```

Comment:
you need to add aria-disabled="true" (string) here, if it's disabled. This will apply the cursor automatically.